index.js

```diff
@@ -105,8 +115,12 @@ exports.fromMapFileComment = function (comment, dir) {
 };
 
 // Finds last sourcemap comment in file or returns null if none was found
-exports.fromSource = function (content) {
-  var m = content.match(commentRx);
+exports.fromSource = function (content, largeSource) {
+  var m;
+
+  if (largeSource) return convertFromLargeSource(content)
+  else m = content.match(commentRx);
```

Comment:
Don't need else here.
Put the `if ... return` statement as first line (early exit).

Then continue with `var m = ....`.
That reads a lot clearer.